src/test/regress/expected/isolation_get_all_active_transactions.out

```diff
@@ -49,26 +49,35 @@ count
 (1 row)
 
 step s3-as-user-1:
- -- User should only be able to see its own transactions
- SET ROLE test_user_1;
- SELECT count(*) FROM get_all_active_transactions() WHERE transaction_number != 0;
- SELECT count(*) FROM get_global_active_transactions() WHERE transaction_number != 0;
+	-- Eventhough we change the user via SET ROLE, the backends' (e.g., s1/2-begin-insert)
```

Comment:
It was buggy that we showed properly on this test. Even pg_stat_activity cannot work. `SET ROLE` (or others) cannot change the backends `RoleId`. We need to re-connect but that doesn't work on isolation tests.

```SQL
[local] onderkalaci@postgres:5432-17026=# select usename from pg_stat_activity where pid = pg_backend_pid();
┌─────────────┐
│   usename   │
├─────────────┤
│ onderkalaci │
└─────────────┘
(1 row)

Time: 3.512 ms
 SET ROLE test_readonly ;
SET
Time: 1.430 ms
[local] onderkalaci@postgres:5432-17026=> select usename from pg_stat_activity where pid = pg_backend_pid();
┌─────────────┐
│   usename   │
├─────────────┤
│ onderkalaci │
└─────────────┘
(1 row)

Time: 1.859 ms
```

Reply:
> It was buggy that we showed properly on this test.


The bug was that use used `distTx` to set the `myBackend->roleId`.  But roles cannot be changed like that